src/hotspot/cpu/aarch64/nativeInst_aarch64.cpp

```diff
@@ -526,21 +529,22 @@ void NativeCallTrampolineStub::set_destination(address new_destination) {
 
 // Generate a trampoline for a branch to dest.  If there's no need for a
 // trampoline, simply patch the call directly to dest.
-address NativeCall::trampoline_jump(CodeBuffer &cbuf, address dest) {
+void NativeCall::trampoline_jump(CodeBuffer &cbuf, address dest, JVMCI_TRAPS) {
```

Comment:
In the original changeset, this hunk has the `#if INCLUDE_JVMCI` addition. Plus `#endif` later.

Reply:
Thanks for spotting this!  Fixed.